modules/serverless-create-trigger-odc.adoc

```diff
@@ -1,33 +1,34 @@
 // Module included in the following assemblies:
 //
-// * /serverless/knative_eventing/serverless-using-brokers.adoc
+// * /serverless/develop/serverless-triggers.adoc
 
 :_content-type: PROCEDURE
 [id="serverless-create-trigger-odc_{context}"]
-= Creating a trigger using the Developer perspective
+= Creating a trigger by using the web console
 
-After you have created a broker, you can create a trigger in the web console *Developer* perspective.
+After you have created a broker, you can create a trigger by using the web console.
 
 .Prerequisites
 
 * The {ServerlessOperatorName}, Knative Serving, and Knative Eventing are installed on your {product-title} cluster.
 * You have logged in to the web console.
-* You are in the *Developer* perspective.
 * You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
 * You have created a broker and a Knative service or other event sink to connect to the trigger.
 
 .Procedure
 
 . In the *Developer* perspective, navigate to the *Topology* page.
 . Hover over the broker that you want to create a trigger for, and drag the arrow. The *Add Trigger* option is displayed.
-+
-image::add-trigger-odc.png[Create a trigger for the broker]
 . Click *Add Trigger*.
 . Select your sink as a *Subscriber* from the drop-down list.
-+
 . Click *Add*.
 
 .Verification
-* After the subscription has been created, it is represented as a line that connects the broker to the service in the *Topology* view:
-+
-image::verify-trigger-odc.png[Trigger in the Topology view]
+
+* After the subscription has been created, you can view it represented as a line that connects the broker to the event sink in the *Topology* view.
```

Comment:
If it doesn't change the meaning, you might consider front-loading the object they are acting on (Topology view): 
```suggestion
* After the subscription has been created, you can view it in the *Topology* view, where it is represented as a line that connects the broker to the event sink.
```

Reply:
Great suggestion, thank you!